src/mailchimp/mailchimp_manager.py

```diff
@@ -99,7 +125,38 @@ def upload_certificates(self, attendees: AttendeeManager,
         Raises:
             TODO: Document potential errors.
         """
-        raise NotImplementedError
+	from mailchimp_marketing import Client
+        
+        mailchimp = Client()
+        mailchimp.set_config(self._keys)
+
+        folder_id = self._folder_id
+
+
+        operations = []
+        for attendee in attendees:
+                pdf_file = attendee.get_file()
+                with open(pdf_file, "rb") as pdf_file:
+                        self._base64_file = base64.b64encode(pdf_file.read())
+                operation = {
+                        "method": "POST",
+                        "path": f"/file-manager/files",
+                        "operation_id": str(attendee.get_id()),  #if there is an attribute to the attendee object for an id or else a name
+                        "body":({
+                                "name"  : attendee.get_Fname() + '.pdf',
+                                "file_data": self._bas64_file
+                                "folder_id": self._folder_id
+                                })
+                        }
```

Comment:
Make sure to stringify the body using the `json` module, as per the Mailchimp API example.
It is possible that multiple attendees with have the same first name. The basename of `file_path` in the `Attendee` object should suffice as the filename for Mailchimp.